### ISSUE TITLE
Screen Reader Announces "Enter to Sort" Despite Sorting Being Disabled

### ISSUE THREAD

**Issue by ghulamghousdev** [bryntum/support#11929]
[Forum post](https://forum.bryntum.com/viewtopic.php?f=51&t=34101&p=173896#p173896)

A. Sorting has been disabled for all columns using sort: false within the feature configuration. However, the screen reader still announces "Enter to sort," which is misleading and should not occur.

B. Additionally, when focus moves to Name items that include an accordion icon, the screen reader again announces "Enter to sort," even though sorting is not enabled. This behavior is incorrect and should be addressed.

Example:
https://bryntum.com/products/gantt/examples/advanced/
![sortingFalse_AccordionAnnouncement.png](https://forum.bryntum.com/download/file.php?id=36446)
![sortingFalse_HeaderAnnouncement.png](https://forum.bryntum.com/download/file.php?id=36445)

**Comment by amurashincertinia** [comment]
Is this related to #12943?

**Comment by matsbryntse-dev** [comment]
Yes dupe!
